test/ui-2/test/clear_and_delete_step.test.js

```diff
@@ -21,49 +20,44 @@ describe('Clear and Delete Steps', () => {
   }, timeout);
 
   test('Brightness Module is added', async () => {
-
     await page.click('[data-value=\'brightness\']');
     const Length2 = await page.evaluate(() => document.querySelectorAll('.step').length);
     expect(Length2).toBe(3);
   }, timeout);
 
   test('Contrast Module is added', async () => {
-
     await page.click('[data-value=\'contrast\']');
     const Length3 = await page.evaluate(() => document.querySelectorAll('.step').length);
     expect(Length3).toBe(4);
   }, timeout);
 
   test('Saturation Module is added', async () => {
-
     await page.click('[data-value=\'saturation\']');
     const Length4 = await page.evaluate(() => document.querySelectorAll('.step').length);
     expect(Length4).toBe(5);
   }, timeout);
 
   test('Rotate Module is added', async () => {
-
     await page.click('[data-value=\'rotate\']');
     const Length5 = await page.evaluate(() => document.querySelectorAll('.step').length);
     expect(Length5).toBe(6);
   }, timeout);
 
   test('Crop Module is added', async () => {
-
     await page.click('[data-value=\'crop\']');
     const Length6 = await page.evaluate(() => document.querySelectorAll('.step').length);
     expect(Length6).toBe(7);
   }, timeout);
 
   test('Delete step is pressed', async () => {
-
-    await page.evaluate(() => { document.querySelectorAll('.remove')[1].click(); });
+    await page.evaluate(`(async() => {
```

Comment:
OK, i refactored this line into a string instead of a function. But, aren't we doing an awful lot of `evaluate()` in this test? Is this required because of the asynchronous nature of the code?